--- a/src/Queue/RabbitMQQueue.php
+++ b/src/Queue/RabbitMQQueue.php
@@ -62,14 +62,14 @@
     /**
      * Holds the Configuration
      */
-    protected QueueConfig $config;
+    protected QueueConfig $queueConfig;
 
     /**
      * RabbitMQQueue constructor.
      */
     public function __construct(QueueConfig $config)
     {
-        $this->config = $config;
+        $this->queueConfig = $config;
         $this->dispatchAfterCommit = $config->isDispatchAfterCommit();
     }
 
@@ -735,9 +735,9 @@
         return [$destination, $exchange, $exchangeType, $attempts];
     }
 
-    protected function getConfig(): QueueConfig
+    public function getConfig(): QueueConfig
     {
-        return $this->config;
+        return $this->queueConfig;
     }
 
     /**
